pilz_extensions/package.xml

```diff
@@ -19,6 +19,7 @@
   <url type="repository">https://github.com/PilzDE/pilz_industrial_motion</url>
 
   <build_depend>clang-tidy</build_depend>
+  <build_depend>libclang-dev</build_depend>
```

Comment:
Is this for running `clang-tidy`? Should it then not be a `test_depend`?

Reply:
clang-tidy is run during build:
https://github.com/PilzDE/pilz_industrial_motion/blob/c285668520cd8c165d7927c109c03597e4c5f354/pilz_extensions/CMakeLists.txt#L23

Reply:
Hm. That's a rather heavy dependency just for building the packages.

Reply:
I opened https://github.com/PilzDE/pilz_robots/pull/239 on https://github.com/PilzDE/pilz_robots which seems more reasonable and should be sufficient to keep a clean and tidy code base.